tests/func/test_data_cloud.py

```diff
@@ -395,6 +430,50 @@ def _get_cloud_class(self):
         return RemoteSSH
 
 
+here = os.path.abspath(os.path.dirname(__file__))
+ssh_user = "user"
+ssh_key_path = os.path.join(here, "{0}.key".format(ssh_user))
+
+
+@pytest.yield_fixture()
+def server():
```

Comment:
we already have this (or actually `ssh()` fixture that I've introduced instead) fixture in `tests/unit/remote/ssh`, no need to copypaste it.

Reply:
Cool. I moved the `ssh()` to conftest.py